assets/marketplace/resources/itemlist.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
this file is not needed now? please remove it.

Reply:
Sure, I will do code clean up once PR is totally ready.